api/src/routers/Base.router.ts

```diff
@@ -43,20 +44,22 @@ abstract class BaseRouter {
     this._basePath = basePath;
     this._router = Router();
     this._controller = controller;
-    console.log(`⚡ [Server]: {Router} :: ${this.constructor.name} initialized...`);
```

Comment:
Can we change this to be
`this._logger = logger`

and also add the property above with the correct type?